CHANGELOG.md

```diff
@@ -10,17 +10,17 @@
 
   ([PR #1326](https://github.com/alphagov/govuk-frontend/pull/1326))
 
-- Update form inputs to use new WCAG 2.1 compliant focus style
+- Update accordion to use new WCAG 2.1 compliant focus style
 
   To migrate: [TODO add migration instructions before we ship v3.0.0]
 
-  ([PR #1312](https://github.com/alphagov/govuk-frontend/pull/1312))
+  ([PR #1324](https://github.com/alphagov/govuk-frontend/pull/1324))
 
-- Update footer links to use new focus style
```

Comment:
I think this has got lost in the rebase?

Reply:
Thanks, I had a bit of a mare in the rebase on this one